package-lock.json

```diff
@@ -70,10 +71,11 @@
       }
     },
     "node_modules/@azure-tools/cadl-ranch-api": {
-      "version": "0.3.0",
-      "resolved": "https://registry.npmjs.org/@azure-tools/cadl-ranch-api/-/cadl-ranch-api-0.3.0.tgz",
-      "integrity": "sha512-ZnwHxCbosJUNRsdVE+8b6RZlFCG8TaPQXkzK0cM4k6lgfaSaYqTrvBPra4SSBgZkV+wxHpQQx3RGmqcMJ6vNcg==",
+      "version": "0.3.1",
```

Comment:
I don't think these package version bumps are intended for this PR.  We just wanted to remove typespec-autorest